master/internal/core.go

```diff
@@ -595,7 +595,7 @@ func (m *Master) Run() error {
 	)
 	template.RegisterAPIHandler(m.echo, m.db, authFuncs...)
 
-	m.system.Tell(m.rp, sproto.ConfigureEndpoints{System: m.system, Echo: m.echo})
+	m.system.Ask(m.rp, sproto.ConfigureEndpoints{System: m.system, Echo: m.echo})
```

Comment:
blocking: There needs to be a `.Get()` to actually wait for the other actor.

non-blocking: A comment about why this is an ask would be helpful.

Reply:
> blocking: There needs to be a .Get() to actually wait for the other actor

Doh.  Been a while since I wrote actor code.